AutoTransformPy/mirror.py

```diff
@@ -18,3 +31,35 @@ def mirror(image_path, direction = 'all'):
     mirrored_images: np.array
     """
 
+    # check for valid input parameters
+
+    if not isinstance(image_path, str):
+        raise TypeError("The file path must be a string.")
+
+    if not isinstance(direction, str):
+        raise TypeError("The direction must be a string: 'horizontal', 'vertical', or 'all'")
+
+    if not direction in ["horizontal","vertical", "all"]:
```

Comment:
Might be reasonable to use lower(direction) here so that users can type "Horizontal" or "All"

Reply:
Good catch! Added `lower()` to the `direction` argument in `mirror.py` in pull request # 26